src/actions/guides/frontend/rendering_html.cr

```diff
@@ -579,6 +579,25 @@ class Guides::Frontend::RenderingHtml < GuideAction
     # => "about a month"
     ```
 
+    ### Cycle values
+
+    The most common case is alternating an HTML class name between rows of data. Lucky comes with a `cycle` method that makes this much
+    easier to do.
+
+    ```crystal
+    posts.each do |post|
+      tr class: cycle(["bg-gray-600", ""], "") do
+        td post.title
+      end
+    end
+    ```
+
+    In this example, the first row, and all even rows, will be `<tr class="bg-gray-600">`, but the next row, and all odd rows, will be
```

Comment:
```suggestion
    In this example, the first row, and all odd rows, will be `<tr class="bg-gray-600">`, but the second row, and all even rows, will be
```
Not sure how it works, but I think this is correct?
1 - "bg-gray-600"
2 - ""
3 - "bg-gray-600"
4 - ""

Reply:
Yeah, that's correct. I guess I was basing it off the index (starting with 0).... 
0 - "bg-gray-600"
1 - ""
2 - "bg-gray-600"
3 - ""

😅  very programmer of me. But maybe your suggestion makes more sense